utils/detect_revision.py

```diff
@@ -11,6 +11,12 @@
 import re
 import string
 
+# The name of the next stable release. We use semantic version strings in the form
+# "1.0~..." where "1.0" is the next release. Update this string after every release,
+# and also if we have to rename the next target during the release cycle.
+# See https://github.com/widelands/widelands/issues/3979 for a full discussion.
+next_stable_version = '1.0'
```

Comment:
How about we put this in a configuration file, just like we do with `WL_RELEASE`?

Reply:
Done